platform/windows/os_windows.cpp

```diff
@@ -2216,6 +2226,9 @@ void OS_Windows::set_window_fullscreen(bool p_enabled) {
 
 		MoveWindow(hWnd, pos.x, pos.y, size.width, size.height, TRUE);
 
+		SystemParametersInfoA(SPI_GETMOUSETRAILS, 0, &restore_mouse_trails, 0);
+		if (restore_mouse_trails > 0)
```

Comment:
BTW, Windows docs say:

> Determines whether the Mouse Trails feature is enabled. This feature improves the visibility of mouse cursor movements by briefly showing a trail of cursors and quickly erasing them.
>
> The pvParam parameter must point to an integer variable that receives a value. if the value is zero or 1, the feature is disabled. If the value is greater than 1, the feature is enabled and the value indicates the number of cursors drawn in the trail. The uiParam parameter is not used.

https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-systemparametersinfoa

So shouldn't this be `> 1`?